app/models/educator.rb

```diff
@@ -37,6 +37,12 @@ def as_json(options = {})
     super(options.merge({ except: [:student_searchbar_json] }))
   end
 
+  def lock_access!(opts = { })
```

Comment:
Override the `lock_access!` method provided by Devise in order to add an email notification to developers after user access is locked.

Original method defined here: 

https://github.com/plataformatec/devise/blob/7b3081760f32e7150601776276bd809601b8eaaa/lib/devise/models/lockable.rb#L42

Reply:
👍 want to put that right in as a comment?